scripts/audit-run

```diff
@@ -3,29 +3,26 @@
 : ${FIREWALLA_HOME:=/home/pi/firewalla}
 source ${FIREWALLA_HOME}/platform/platform.sh
 
-logger 'FIREWALLA.ACLAUDIT Start'
+logger 'FIREWALLA.ACLAUDIT Starting'
 
 sudo mkdir -p /alog/
 sudo rm -r -f /alog/*
-sudo umount -l /alog
-sudo mount -t tmpfs -o size=10m tmpfs /alog
-
-sleep 3
-
-sudo cp ${FIREWALLA_HOME}/etc/rsyslog.d/30-acl-audit.conf /etc/rsyslog.d/
-sudo cp ${FIREWALLA_HOME}/etc/logrotate.d/acl-audit /etc/logrotate.d/
-sudo systemctl restart rsyslog
-sudo iptables -I FW_DROP -j LOG --log-prefix "[FW_ACL_AUDIT]"
-sudo ip6tables -I FW_DROP -j LOG --log-prefix "[FW_ACL_AUDIT]"
 
 # ensure log file is readable
-sudo touch /alog/acl-audit.log
+sudo mkfifo /alog/acl-audit.log
 sudo chgrp adm /alog/acl-audit.log
 sudo chown syslog /alog/acl-audit.log
 sudo chmod 644 /alog/acl-audit.log
 
 # sudo ln -s /alog/acl-audit.log /var/log/acl-audit.log
 sudo mkdir -p /log/alog
-sudo ln -s /alog/acl-audit.log /log/alog/acl-audit.log
+sudo ln -sf /alog/acl-audit.log /log/alog/acl-audit.log
+
+
+sudo cp ${FIREWALLA_HOME}/etc/rsyslog.d/30-acl-audit.conf /etc/rsyslog.d/
+sudo systemctl restart rsyslog
+
+sudo iptables -C FW_DROP -j LOG --log-prefix "[FW_ACL_AUDIT]" &>/dev/null || sudo iptables -I FW_DROP -j LOG --log-prefix "[FW_ACL_AUDIT]"
```

Comment:
better to move this iptables commands to control/install_iptables_setup.sh

Reply:
also the processing of iptables drop logs may slow down red/blue, need a platform on/off func so that we can easily enable/disable in the future